src/lambda_function/function.py

```diff
@@ -0,0 +1,82 @@
+import boto3
+import logging
+import os
+import time
+import json
+import datetime
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+ec2_client = boto3.client('ec2')
+autoscaling_client = boto3.client('autoscaling')
+
+OLDER_THAN_DAYS = os.environ['OLDER_THAN_DAYS']
+
+launch_configurations_dump = autoscaling_client.describe_launch_configurations()
```

Comment:
Do these inside the method(s). The results they provide can change rapidly in some accounts, and doing them at the top level will cache the results across lambda function runs.